third_party/bazel/tf_protobuf/tf_protobuf_workspace.bzl.tpl

```diff
@@ -1,14 +1,18 @@
 load("@org_tensorflow//third_party:repo.bzl", "tf_http_archive", "tf_mirror_urls")
 
 def _tf_protobuf_workspace():
+    # NOTE(lanbo.llb): tf_http_archive will check if rule with same already exists,
+    # if so, the duplicate rule by `tf_http_archive` will not be used.
+    # Our own version of protobuf will be used instead of tf_community's.
+    # Make sure this rule is loaded first when building tf's custom op/passes.
     tf_http_archive(
-        name = "tf_protobuf",
-        patch_file = ["//third_party/bazel/tf_protobuf:protobuf.patch"],
+        name = "com_google_protobuf",
```

Comment:
It seems that it's renamed to overwrite tf's `com_google_protobuf` intentionally, but I didn't get the motivation...

Reply:
Since the `protobuf` we get from github urls already have `bazel` build files, it's workspace is `com_google_protobuf `. If we manually changes this name by adding a patch, it would require some extra works, so I take the easy way to use the same rule name as in `org_tensorflow`, thus we can use the bazel patch/BUILD/bzl files directly from release branch like r2.4.0-rc3/r1.15.0-rc5.